components/admin_console/manage_teams_modal/manage_teams_modal.test.tsx

```diff
@@ -66,16 +81,24 @@ describe('ManageTeamsModal', () => {
                 getTeamsForUser,
             },
         };
+        const intlProviderProps = {
+            defaultLocale: 'en',
+            locale: 'en',
+            messages: {'test.value': 'Actual value'},
+        };
 
-        const wrapper = shallow(
-            <ManageTeamsModal {...props}/>,
-        );
-
-        process.nextTick(() => {
-            expect(wrapper.state('teams')).toEqual([mockTeamData]);
-            expect(wrapper.state('teamMembers')).toEqual([{team_id: '123test'}]);
-            expect(wrapper).toMatchSnapshot();
-            done();
+        let wrapper: ReactWrapper<any>;
+        await act(async () => {
+            wrapper = mount(
+                <IntlProvider {...intlProviderProps}>
+                    <ManageTeamsModal {...props}/>
+                </IntlProvider>,
+            );
         });
+        wrapper!.update();
+
+        expect(wrapper!.find('.manage-teams__team-name').text()).toEqual(mockTeamData.display_name);
+        expect(wrapper!.find(ManageTeamsDropdown).props().teamMember).toEqual({team_id: '123test'});
+        expect(wrapper!).toMatchSnapshot();
```

Comment:
As I changed the component from shallow to mount, matching snapshot is failing due to big size of snapshot (I assume). Is there any suggestion how can I fix that?

Reply:
Any guidance on this point @devinbinnie ?

Reply:
@devinbinnie I already updated the snapshot. The issue here is that matching with snapshot is taking too long and the test is failing. I believe the reason is that I changed from `shallow` to `mount`.

Reply:
@sinansonmez oh i got it yes jest doesnt like too long snapshots.